code/game/objects/items/devices/radio/radio.dm

```diff
@@ -516,10 +516,6 @@
 					recalculateChannels()
 				. = TRUE
 
-/obj/item/radio/suicide_act(mob/living/user)
```

Comment:
what's going on

Reply:
Whoops, meant to mention that.

Reply:
tldr; duped suicide_act I saw while editing the file. Now mentioned in body.